docs/log-ingestion/logstash/logstash-splunk.md

```diff
@@ -1,9 +1,9 @@
 # Splunk
 
-- Siglens can be ingested data in Splunk format using Logstash through http plugin. Install Logstash. Read more about installation [here](#install-logstash).
-- Make sure that the `hosts` in the configuration has the `/splunk` suffix.
+- Siglens can be ingested data in Splunk format using Logstash through http plugin. Read more about installation [here](#install-logstash).
+- Make sure that the `hosts` in the configuration should be the Splunk HEC even collector endpoint `/splunk/services/collector/event`.
```

Comment:
Line 3:
* Change "can be ingested data" to "can ingest data"
* Change "through http plugin" to "through the http plugin"

Line 4:
* "Make sure ... should be ..." doesn't make sense. You could do "Make sure ... is ..." or "The `hosts` ... should be ... " or something similar